### PR TITLE
Add support for default camera orientation

### PR DESCRIPTION
**Pull request by hultonha**
This change still needs a bit more testing and maybe a couple more comments but it's more or less there, will try and get it merged tomorrow.

image

### CODE REVIEW

**Review by moraaar — APPROVED** [review]
LGTM. But I was wondering why pitch and yaw only for default orientation? It could be a Vector3 so the user could specify initial value for roll too.

**Comment by hultonha** [comment]
> LGTM. But I was wondering why pitch and yaw only for default orientation? It could be a Vector3 so the user could specify initial value for roll too.

The camera does currently support role but it's only in slight weird circumstances like 'Be this camera' when the entity has roll. It's intentionally not included here as it makes control more unwieldy in the general case (if there is a customer need in future we can always add it though 🙂)

File: Code/Editor/EditorPreferencesPageViewportCamera.cpp [hunk 1]
```diff
@@ -77,6 +77,7 @@ void CEditorPreferencesPage_ViewportCamera::Reflect(AZ::SerializeContext& serial
         ->Field("PanInvertedX", &CameraMovementSettings::m_panInvertedX)
         ->Field("PanInvertedY", &CameraMovementSettings::m_panInvertedY)
         ->Field("DefaultPosition", &CameraMovementSettings::m_defaultPosition)
+        ->Field("DefaultOrientation", &CameraMovementSettings::m_defaultPitchYaw)
```
Comment (SergeyAMZN) [inline Code/Editor/EditorPreferencesPageViewportCamera.cpp:80]: Does it need a version bump up?

**Review by SergeyAMZN — APPROVED** [review]

File: Code/Editor/EditorPreferencesPageViewportCamera.cpp (hunk 1 above)
Comment (hultonha) [reply Code/Editor/EditorPreferencesPageViewportCamera.cpp:80]: Ah cool thanks! I'll leave as-is for now 👍

**Pull request merged**